__tests__/app.test.js

```diff
@@ -56,18 +56,170 @@ describe("GET", () => {
       });
   });
 
+  test("/api/articles/ 200: responds with all the articles with the total comment count for each article", () => {
+    return request(app)
+      .get("/api/articles/")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          articlesWithTotalComments: [
+            {
```

Comment:
rather than make an assertion against a massive hard code value, iterate over your articles and make an assertion about each article.  When doing this you should check that you are not getting back an empty array, and also make assertions about the data type stored on each article object's keys.

Reply:
also please write a test check that the articles are sorted by date (hint - use jest sorted)